druid-shell/src/platform/linux.rs

```diff
@@ -0,0 +1,38 @@
+// Copyright 2021 The Druid Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Linux specific extensions.
+use crate::Clipboard;
+
+/// Linux specific extensions to [`Application`]
+///
+/// [`Application`]: crate::Application
+pub trait LinuxApplicationExt {
```

Comment:
Is this  'linux' thing, or only  a gtk thing?

Reply:
this is a Linux thing

Reply:
Is this a Linux thing? The primary selection (I wouldn't really call it "clipboard"...) is AFAIK an X11 thing and not implemented on Wayland. But my knowledge here might be out of date and I am too lazy to dig into Gtk's Wayland backend right now, so I am just leaving this as a comment here...

Reply:
:man_facepalming: I really forgot wayland's existence.

Reply:
there is a [unstable wayland protocol](https://github.com/wayland-project/wayland-protocols/blob/36cee4bdbcaf3a75f567315809953b5e21d703da/unstable/primary-selection/primary-selection-unstable-v1.xml) and is supported by gtk, qt and by kde plasma, gnome and sway, so :man_shrugging: